src/ngraph/op/util/attr_types.hpp

```diff
@@ -257,6 +257,10 @@ namespace ngraph
                 , m_axis(0)
             {
             }
+            AutoBroadcastSpec(const char* type)
```

Comment:
Could we use `std::string` here?

Reply:
I consider it but `std::string` can not be used to implicit conversion here. It doesn't compile.